recipes/openimageio/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(OpenImageIO REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} OpenImageIO::OpenImageIO)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_14)
```

```suggestion
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
```
I'm a dum dum, sorry!